### DISCUSSION TITLE
Overclocking engines

### DISCUSSION THREAD

**Discussion by Zhertva** [FakeFishGames/Barotrauma#5976] [Ideas and suggestions]
Add the possibility to overclock engines over 100% capacity to gain movement speed and to get out of difficult situations or attempt coordinated moves during combat.
Overclocking engines may damage them when performed and start a fire in the engines room is maintained for a too long time.

**Comment by ViktorShield** [comment]
200% is way too much, 50-75% is enough

**Reply by Zhertva** [reply]
Yes of course, i was not talking of getting a +100% speed, but to go over the normal 0-100% engines range, like a 125% (+25% over max speed) or 150% (+50% over max speed)

**Reply by DreamerOfRain** [reply]
Do note that engines already get +100% speed bonus when you reach 100 Helm for captain. This can break a lot of sub designs.

**Comment by ViktorShield** [comment]
I just place a shuttle engine right behind the main engine, one over another. You can on/off those separately using relays and standard pulse-to-static component, so it can be connected to the navigation terminal. The secondary engine has different repair parameters, as well as tweaked force

**Comment by James-E-A** [comment]
Partially implemented in the Ancients update with Mechanic skills

> You can Tinker with an engine, increasing its torque by 150%.

> [Overclocking] items damages them, but the effects of Tinkering are increased by 50%.
